### ISSUE TITLE
Code review

### ISSUE THREAD

**Issue by claudio-moya** [rakuten-rex/stepper#3]
Component
- [x] Fix 1 Accessibility violations
- [x] Add `counter` as prop
- [x] Add `<input type="hidden" value={counter} />` to store the counter value with props like `id`, `name`, `value` (keep the `span` wrapper)
- [x] Add `maxLimit` prop as limit for counter

Stories
- [x] Add `counter`, `maxLimit`, `id`, `name`, `value` to `Dynamic Props`
- [x] Add `disabled` prop in Theme as `Props` category and keep `Theme` category name for theme colors

Docs
- [x] Update docs for readme file in `project-scripts/webpack/markdown`

**Comment by claudio-moya** [comment]
Component
- [ ] In `Default view`, and `Theme` stories the plus icon feature is not working
- [ ] Use `div` instead of `p` tag for the sample wrapper to avoid this warning message
http://localhost:8080/?path=/story/stepper--with-theme-react-and-css-vars
![Screen Shot 2020-01-29 at 11 57 22 AM](https://user-images.githubusercontent.com/10554673/73324554-ad63a780-428e-11ea-9aff-fbf453fb40ed.png)

Theme 
- [ ] Add color option to `plus` and `minus` icon on default state
- [ ] Add background color option for default and disabled state

**Comment by claudio-moya** [comment]
Component
- [ ] Disabled view colors are inverted ?
http://localhost:8080/?path=/story/stepper--disabled-view
- [ ] The classNames for `rex-icon-button plus` and `rex-icon-button minus` are backwards
